app/validation/validator.py

```diff
@@ -26,7 +26,9 @@ def validate_schema(self, json_to_validate):  # noqa: C901  pylint: disable=too-
 
         numeric_answer_ranges = {}
 
-        all_groups = itertools.chain.from_iterable(section.get('groups', []) for section in json_to_validate['sections'])
+        all_groups = []
+        for section in json_to_validate['sections']:
+            all_groups.extend(section.get('groups'))
```

Comment:
Just put this in the below `for` removes duplication

Reply:
Can't do that as it will only have a list of previous groups then, which isn't sufficient for routing